jobs/upload-brew.Jenkinsfile

```diff
@@ -0,0 +1,115 @@
+import org.yaml.snakeyaml.Yaml;
+
+node {
+    checkout scm
+    // these are script global vars
+    pipeutils = load("utils.groovy")
+    pipecfg = pipeutils.load_pipecfg()
+    libcloud = load("libcloud.groovy")
+
+}
+
+def brew_principle = pipecfg.brew.principle
+def brew_profile = pipecfg.brew.profile
+def arches = pipeutils.get_supported_additional_arches()
```

Comment:
We will always have a build with all arches, so I think in this case we can assume use all arches, instead of checking it, WDYT?

Reply:
Ideally we always ship with all arches, but one day there may be a need to exclude one arch when doing a release so it would probably be better to check rather than assume.

Reply:
Hotfix builds is an example where we likely wouldn't have all arches (I'm not sure if the Brew job would be used in that case, but it should do the right thing if it were).

Reply:
Ok, how about we let all arches as default,  but an option to overwrite it?

Reply:
I think if the goal is to eventually make it part of the release job, then we should just match the same semantics as that. I.e. let's have an `ADDITIONAL_ARCHES` and a `ALLOW_MISSING_ARCHES` parameter like in the release job?